test/periphery/PixCashierUpgradeable.test.ts

```diff
@@ -121,9 +135,9 @@ describe("Contract 'PixCashierUpgradeable'", async () => {
 
     it("Emits the correct event", async () => {
       await proveTx(brlcMock.mint(cashierClient.address, tokenAmount));
-      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount))
+      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount, TRANSACTION_ID))
         .to.emit(pixCashier, "CashOut")
-        .withArgs(cashierClient.address, tokenAmount, tokenAmount);
+        .withArgs(cashierClient.address, tokenAmount, tokenAmount, TRANSACTION_ID);
```

Comment:
Codacy has a fix for the issue: [Replace `cashierClient.address,·tokenAmount,·tokenAmount,·TRANSACTION_ID` with `⏎··········cashierClient.address,⏎··········tokenAmount,⏎··········tokenAmount,⏎··········TRANSACTION_ID⏎········`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9795032)
```suggestion
        .withArgs(
```